pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -273,11 +290,14 @@ def __init__(self,
         :param Optional[List[Callable]] transformations: Optional list of transformations to apply to
                resources that will be created by this chart prior to creation. Allows customization of the
                chart behaviour without directly modifying the chart itself.
+        :param Optional[str] resource_prefix: An optional prefix for the auto-generated resource names.
+               Example: A resource created with resource_prefix="foo" would produce a resource named "foo-resourceName".
```

Comment:
Is there any way we can minimize the repetition here? We describe the `resource_prefix` param in this `helm.py` 4 times.

Reply:
I created a base class to cut down on the repetition slightly.